ArjunaJTA/jta/src/test/resources/arquillian.xml

```diff
@@ -24,14 +24,16 @@
 
    <container qualifier="jboss" default="true">
         <configuration>
-            <!-- defaults to JBOSS_HOME
-<property name="jbossHome">$WORKSPACE/jboss-as/build/target/wildfly-$WILDFLY_MAIN_VERSION</property> -->
-
-            <property name="serverConfig">standalone.xml</property>
+            <property name="serverConfig">standalone-full.xml</property>
```

Comment:
Why do we need here the standalone-full.xml?

Reply:
`standalone.xml` might work as well (I didn't test it). If you prefer to use `standalone.xml` I can try it out. Anyway, the choice between `standalone.xml` and `standalone-full.xml` wasn't the problem that this PR fixed. A CMR configuration was needed.

Reply:
I tested it and it works as well. I saw the CMR configuration you added, thank you for that!

Reply:
So do you want me to use `standalone.xml` instead of `standalone-full.xml`?

Reply:
Actually I don't know. I was only looking for a reason for this change.

Reply:
If the problem being fixed by the PR is the CMR config, why is the PR as big as it is?

Reply:
Yep, that's the reason. I needed to add an extra test to check we use CMR records. Plus, extra code was needed to configure the WildFly instance started by Arquillian

Reply:
> Actually I don't know. I was only looking for a reason for this change.

I'll change this to `standalone.xml` :-)